projects/1.20/assets/biomes-o-plenty-delight/biomesoplentydelight/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "item.biomesoplentydelight.cooked_bramble": "煮熟的荆条",
+  "item.biomesoplentydelight.barley_bread": "大麦面包",
+  "item.biomesoplentydelight.flesh": "肉块",
+  "item.biomesoplentydelight.glowworm_ice_cream": "萤火虫冰淇淋",

```

```suggestion
  "item.biomesoplentydelight.glowworm_ice_cream": "荧光蛛冰淇淋",
```
经检验，配方中均包含“荧光蛛丝”这一物品，建议全部将“萤火虫”改为“**荧**光蛛”，后方不再一一指出